srcpkgs/foo2zjs/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'foo2zjs'
+pkgname=foo2zjs
+version=2017.12.02
+revision=1
+#wrksrc=
+#create_wrksrc=yes
+#only_for_archs="i686 x86_64"
+build_style=fetch
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends=""
+makedepends=""
+depends=""
+short_desc="Collection of print drivers by rkkda.com"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="GPL-2"
```

Comment:
pls use [SPDX-Identifier ](https://spdx.org/licenses/)

Reply:
My bad, this template isn't quite done yet, I must've merged it in by accident at some point